lib/handlebars/helpers/languageLink.test.js

```diff
@@ -0,0 +1,91 @@
+const handlebars = require('handlebars')
+
+const mockTranslations = {
+  label_lang_en_sv: 'English',
+  label_lang_en_en: 'Engelska',
+  label_lang_sv_sv: 'Swedish',
+  label_lang_sv_en: 'Svenska',
+  label_custom_sv: 'Anpassad',
+  label_custom_en: 'Custom',
+  label_locale_select_link_title_sv: 'Show translation',
+  label_locale_select_link_title_en: 'Visa översättning',
+  label_button_close_sv: 'Stäng',
+  label_button_close_en: 'Close',
+  label_not_translated_en: 'Den här sidan är inte översatt',
+  label_not_translated_sv: 'This page is not translated',
+  label_start_page_sv: 'Start page in English',
+  label_start_page_en: 'Startsida på svenska',
+}
+
+jest.mock('kth-node-i18n', () => ({
+  message: (key, lang) => mockTranslations[`${key}_${lang}`],
```

Comment:
Might be nice if the format of the mock-labels where more like the ones we use in apps, ie `mockTranslations[lang][key]`

Reply:
Yes! Of course. Good suggestion. Much cleaner. ✨